core/math/a_star.cpp

```diff
@@ -234,6 +236,10 @@ Vector3 AStar::get_closest_position_in_segment(const Vector3 &p_point) const {
 
 	for (const Set<Segment>::Element *E = segments.front(); E; E = E->next()) {
 
+		if (!(E->get().from_point->enabled && E->get().from_point->enabled)) {
```

Comment:
Surely, you meant to check both `from_point` and `to_point`, instead of checking `from_point` twice?

Reply:
Yeah, I did. Thanks for catching that. Whenever I did the git rebase to squash my commits it ended up pushing the older commits too (i.e. not part of my work), not sure how to get rid of those.